packages/blade/src/components/List/ListItem.tsx

```diff
@@ -111,12 +111,13 @@ const _ListItem = ({
     if (
       typeof child === 'string' ||
       isValidAllowedChildren(child, 'ListItemLink') ||
+      isValidAllowedChildren(child, 'ListItemText') ||
       isValidAllowedChildren(child, 'ListItemCode')
```

Comment:
Can we put all these ids in a separate file? Same how we are doing in other components.